libs/books/data-access/src/lib/+state/reading-list.reducer.ts

```diff
@@ -18,40 +18,46 @@ export interface ReadingListPartialState {
 export const readingListAdapter: EntityAdapter<ReadingListItem> = createEntityAdapter<
   ReadingListItem
 >({
-  selectId: item => item.bookId
+  selectId: (item) => item.bookId,
```

Comment:
Avoid such unnecessary changes while formatting code.
Applicable for this file.

Reply:
Changed formatting